doc/source/math/basic.rst

```diff
@@ -14,14 +14,6 @@ Fundamental Constants
 There is a bound variable called CONSTANT which contains some basic fundamental
 constants about the universe that you may find handy in your math operations.
 
-.. versionadded:: 0.18
-    Prior to kOS version 0.18, ``constant`` was a function call, and
-    therefore to say ``constant:pi``, you had to say ``constant():pi``.
-    The function call ``constant()`` still exists and still works, but
-    the new way without the parentheses is preferred going forward,
-    and the way with the parentheses may become deprecated later.
-    For the moment, both ways of doing it work.
-
```

Comment:
Agree/Disagree:

I think mentioning that both ways work is still important.  Even if we remove all talk about the version numbers.
We can probably get rid of talk about "for the moment" as well.  This is likely to just permantly stay this way where both ways continue to work.  Just something that says something along these lines: "you may see scripts written that have parentheses on the word "constant", like so: ``constant():pi``.  This is an older syntax that still works, although leaving the parentheses off is now the preferred way."

Reply:
I wrote up something similar to what you described but included the prior-to for those who find older scripts.